README_CN.md

```diff
@@ -48,3 +48,31 @@ startmonster.py 会循环调用，并把 common_params 和 testcase指定的para
 3. 运行startmoster.py
 
 ![image](./image/startmonster.png)
+
+1.注意
+
+在运行前需要打开eosapi/httpapi/http_client.py,在```class HttpClient(object):```中初始化的时候设置指定节点的url，
+例如：self.node_url = 'http://127.0.0.1:8888' 
```

Comment:
please add the node_url as a paramater into the HttpClient() init function. We should NOT change the basic function for erverytime.